src/@types/cozy-client.d.ts

```diff
@@ -91,6 +93,11 @@ declare module 'cozy-client' {
 
   export const useClient = (): CozyClient => CozyClient as CozyClient
 
+  export interface SplitFilenameResult {
```

Comment:
This type seems to be easy to add to cozy-client project here: https://github.com/cozy/cozy-client/blob/a2a811593b4181f559583c994a91d95cf3f7e289/packages/cozy-client/src/models/file.js#L22-L23

When we can do it easily it is preferable to update cozy-client